draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -960,21 +960,37 @@ the SD-JWT.
 
 ## Selectively-Disclosable Validity Claims {#sd-validity-claims}
 
-Claims controlling the validity of the SD-JWT, such as `nbf`, `iat`, and `exp`,
-are usually included in plaintext in the SD-JWT payload, but MAY be
-selectively disclosable instead. In this case, however, it is up to the Holder
-to release the claims to the Verifier. A malicious Holder may try to hide, for
+An Issuer MUST NOT allow any security-critical claim to be selectively disclosable.
+The exact list of such claims will depend on the application,
+and SHOULD be listed by any application-specific profile of SD-JWT.
+The following is a list of standard claim names that SHOULD be considered as
+security-critical by any SD-JWT Issuer:
+
+* `iss` (Issuer)
+* `aud` (Audience), although issuers may want to allow individual entries in the array to be selectively disclosable
+* `exp` (Expiration Time)
+* `nbf` (Not Before)
+* `iat` (Issued At)
+* `jti` (JWT ID)
```

Comment:
I am not sure about the format of this... SD-JWT is payload agnostic, so there is no guarantee the payload uses JWT claims (though it is of course a convention) - for example, W3C VCDM might use their own claims (`issuer`, `validFrom`, etc.)

Reply:
Maybe an explicit reference to this being the default JWT claims would make this clear.